modules/globebrowsing/src/renderableglobe.cpp

```diff
@@ -221,7 +219,7 @@ namespace {
         "ZFightingPercentage",
         "Z-Fighting Percentage",
         "The percentage of the correct distance to the surface being shadowed. Possible "
-        "values: [0.0, 1.0].",
+        "values: [0.0, 1.0].", // @TODO (2024-04-26,  emmbr) clarify. I don't get it
```

Comment:
Someone help, plz

Reply:
Its a tweak value that Jonathas added to fix an issue with Z-fighting.  Iirc its a correction term to the FragDepth when doing rendering the shadow volume.

Not sure that clears things up as I don't understand it either

Reply:
I can't figure out what difference it makes (have tried changing it but see no visual difference on the rendered shadows), so maybe I'll just leave it as is then?